README.md

```diff
@@ -18,3 +18,7 @@ See project [UIREQ](https://issues.folio.org/browse/UIREQ)
 at the [FOLIO issue tracker](https://dev.folio.org/guidelines/issue-tracker).
 
 Other FOLIO Developer documentation is at [dev.folio.org](https://dev.folio.org/)
+
+## BREAKING CHANGES
+
+In adding title-level request we removed support for interfaces that only handle item-level requests.
```

Comment:
Looking carefully at my last comment, I see I asked to update the readme with breaking change info. I meant the CHANGELOG; sorry!

Reply:
I can't fully understand how it should looks in `CHANGELOG`. Is it ok if I'll do something like this:

* BREAKING CHANGE: TLR - data migration for request list, view, create, edit, duplicate. Refs UIREQ-664.
* BREAKING CHANGE: TLR - data migration for dialogboxes and reorder list. Refs UIREQ-665.
* BREAKING CHANGE: TLR - data migration for hold shelf clearance report. Refs UIREQ-677.
* BREAKING CHANGE: TLR - data migration, move `holdingsRecordId`. Refs UIREQ-685.

Because now we don't have any `BREAKING CHANGE` section, and in my opinion it's not the best way if add it in the middle of changelog.
Could you please clarify?
Thanks!

Reply:
done, changed to:
BREAKING: TLR - data migration for request list, view, create, edit, duplicate. Refs UIREQ-664.
BREAKING: TLR - data migration for dialogboxes and reorder list. Refs UIREQ-665.
BREAKING: TLR - data migration for hold shelf clearance report. Refs UIREQ-677.
BREAKING: TLR - data migration, move `holdingsRecordId`. Refs UIREQ-685.